user/templates/register.html

```diff
@@ -50,14 +50,14 @@ <h1>Register</h1>
                         {% endif %}
                         <div class="form-group">
                             <label for="id_phone">Phone</label>
-                            <input type="text" name="phone" maxlength="225" class="form-control" placeholder="070-000 00 00" id="id_phone" {% if form.phone %}value="{{ form.phone }}"{% endif %}>
+							<input type="text" name="phone" maxlength="225" class="form-control" placeholder="070-000 00 00" id="id_phone" {% if form.phone %}value="{{ form.phone }}"{% elif request.user.phone %}value="{{request.user.phone}}"{% endif %}>
                         </div>
                         <div class="form-row mt-3">
                             <div class="col-md-4">
                                 <label for="id_university">University <small>(required)</small></label>
                                 <select name="university" class="form-control" id="id_university">
                                     {% for university in const_universities %}
-                                        <option value="{{ forloop.counter0 }}" {% if form.university == forloop.counter0|stringformat:"i" %}selected=""{% elif not form.university and university == "KTH Royal Institute of Technology" %}selected=""{% endif %}>{{ university }}</option>
+									<option value="{{ forloop.counter0 }}" {% if form.university == forloop.counter0|stringformat:"i" %}selected=""{% elif not form.university and request.user.university == university %}selected=""{% elif not form.university and not request.user.university and university == "KTH Royal Institute of Technology" %}selected=""{% endif %}>{{ university }}</option>
```

Comment:
This needs a bit of fixing, there's three cases which do not seem to match this change.
1. If the form provides `university` use that.
2. If the form does not provide it but the user has it use that.
3. Alternatively use KTH.

Reply:
Actually, this is literally what you are doing here, nice!